nemo/collections/tts/torch/data.py

```diff
@@ -102,6 +102,10 @@ def __init__(
         n_mels: int = 80,
         lowfreq: int = 0,
         highfreq: Optional[int] = None,
+        segment_max_duration: Optional[int] = None,
+        pitch_augment: bool = False,
+        cache_pitch_augment: bool = True,
+        pad_multiple: int = 1,
```

Comment:
Please add info about these to the docstring

Reply:
docstring updated.